packages/html/src/skeleton/skeleton.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { classNames } from '../misc';
+
+export const SKELETON_CLASSNAME = `k-skeleton`;
+
+const states = [];
+
+const options = {};
+
+export type KendoSkeletonProps = {
+    shape?: 'circle' | 'text' | 'rect';
+    animation?: 'wave' | 'pulse';
+    style?: React.CSSProperties;
+};
+
+const defaultProps = {
+    shape: 'text',
+    animation: 'pulse'
+};
+
+export const Skeleton = (
+    props: KendoSkeletonProps & React.HTMLAttributes<HTMLSpanElement>
+) => {
+    const {
+        shape = defaultProps.shape,
+        animation = defaultProps.animation,
+        ...other
+    } = props;
+
+    return (
+        <span
+            {...other}
+            className={classNames(
+                props.className,
+                SKELETON_CLASSNAME,
+                'k-skeleton-',
```

Comment:
Remove this as it results in `k-skeleton-` being rendered as an additional class